analysis_options.yaml

```diff
@@ -45,4 +45,10 @@ linter:
 
     join_return_with_assignment: false
 
-    prefer_const_literals_to_create_immutables: false
\ No newline at end of file
+    prefer_const_literals_to_create_immutables: false
+
+    require_trailing_commas: false
```

Comment:
Doesn't this help with linting so that all the code looks the same? If so This is a bad idea. We have had severe linting issues in the past where a lot of our time was spent reviewing diffs that were due to text spacing in the source code.

Reply:
I'll make the necessary changes and make a separate PR.